src/@next/Icon/icons/ri-flag-line.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 18 19" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@music1353 can you double check the exported file? it seems the size is a bit different 🤔

Reply:
@ninariccimarie thanks adjusted!

Reply:
@music1353 it should probably be the other way around. the size is usually the one with the 0 0 24 24 instead. with the hollow space around the icon.